data/campaigns/Sceptre_of_Fire/scenarios/7_Outriding_the_Outriders.cfg

```diff
@@ -263,7 +263,7 @@
 
                 [message]
                     id=Alanin
-                    message= _ "Oh no, more of them? And there I thought I could catch my breath…"
+                    message= _ "Oh no, more of them? And there I thought I could catch my breath..."
```

Comment:
Wondering if it would be better to reverse this typography rule, and use the ellipsis elsewhere.

Reply:
I said before that I would be okay with the style guide being abolished and letting people just write whatever, let inconsistency reign supreme. I followed your lead in encouraging others to adhere to it. I'll note, however, this was the only instance (aside from another that was disabled-via-comment) in all of Wesnoth text that actually uses the ellipsis character - there are many, many more instances of the three full-stops being used instead. And I also consider that inverting the rule to dictate the use of ellipsis characters may add to the burden of getting writers to follow it (until and unless an automated check enforcing the rules is implemented).

Reply:
I think the reasoning of the style guide is that visually, when rendered in Wesnoth, there is negligible difference between `...` and `…`.

This is an example of three full-stops:
![image](https://github.com/user-attachments/assets/5baa190e-e4a2-4204-9214-f56714524bd3)

And this is the only example of the actual ellipsis character:
![image](https://github.com/user-attachments/assets/6aec24cd-d06a-4047-ac10-60e5601f324d)

Reply:
That makes sense, I'll mark this resolved.